src/index.js

```diff
@@ -1,18 +1,22 @@
 import errorOverlayMiddleware from 'react-dev-utils/errorOverlayMiddleware'
 
+const chunkPathBasic = require.resolve('./entry-basic')
+const chunkPathDevServer = require.resolve('./entry-devserver')
+
 class ErrorOverlayPlugin {
   apply(compiler) {
     const className = this.constructor.name
 
     if (compiler.options.mode !== 'development') return
 
+    const enableDevServer = !!compiler.options.devServer
```

Comment:
`enableDevServer` is an action, please rename it to `devServerEnabled`.

Reply:
Done 👍